packages/test-utils/src/utils/is-focusable.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { focus } from '@wordpress/dom';
+/**
+ * Internal dependencies
+ */
+import './mock-client-rects';
+import getDocument from './get-document';
+
+/**
+ * @param {Element} element
+ * @return {boolean} Whether `element` is focusable
+ */
+export default function isFocusable( element ) {
+	const document = getDocument( element );
+	const focusableElements = focus.focusable.find( document );
```

Comment:
While it would be nice if the `focus` helper could expose an extra utility / option to test a specific node, it might be more efficient to test within the context of `element.parentElement`, rather than the entire document. Not that I expect most of these documents have many elements to begin with in the context of a test...

Reply:
Yeah! I tried to touch other modules as little as possible in this first iteration, but I think it would be better if `focus` provided something like that. I imagine the implementation would be something like `element.matches( SELECTOR ) && isVisible( element )`.

Will change it to `element.parentElement` for now. 👍